agent/util-scripts/gold/test-client-tool-meister/test-61.txt

```diff
@@ -0,0 +1,291 @@
+pbench-tool-meister-start: Error: host: testhost.example.com; tool: mpstat; Run-Failure-Info: mpstat: pbench-sysstat-12.0.3-1 is not installed
+ERROR - "pbench-tool-meister-start --sysinfo='block,security_mitigations,sos' 'lite'" failed to execute successfully (exit code: 21)
+killing /var/tmp/pbench-test-utils/pbench/mock-run/tm/pbench-tool-data-sink.pid (190156)
+--- Finished test-61 test-client-tool-meister (status=1)
++++ pbench tree state
+/var/tmp/pbench-test-utils/pbench
+/var/tmp/pbench-test-utils/pbench/mock-run
+/var/tmp/pbench-test-utils/pbench/mock-run/metadata.log
+/var/tmp/pbench-test-utils/pbench/mock-run/ssh_config
+/var/tmp/pbench-test-utils/pbench/mock-run/ssh_config.d
+/var/tmp/pbench-test-utils/pbench/mock-run/tm
+/var/tmp/pbench-test-utils/pbench/mock-run/tm/pbench-tool-data-sink.err
+/var/tmp/pbench-test-utils/pbench/mock-run/tm/pbench-tool-data-sink.out
+/var/tmp/pbench-test-utils/pbench/mock-run/tm/redis.conf
+/var/tmp/pbench-test-utils/pbench/mock-run/tm/redis.log
+/var/tmp/pbench-test-utils/pbench/mock-run/tm/tm-lite-testhost.example.com.err
+/var/tmp/pbench-test-utils/pbench/mock-run/tm/tm-lite-testhost.example.com.out
+/var/tmp/pbench-test-utils/pbench/mock-run/tm/tm.logs
+/var/tmp/pbench-test-utils/pbench/pbench.log
+/var/tmp/pbench-test-utils/pbench/tmp
+/var/tmp/pbench-test-utils/pbench/tmp/tm-lite-remote-a.example.com.err
+/var/tmp/pbench-test-utils/pbench/tmp/tm-lite-remote-a.example.com.out
+/var/tmp/pbench-test-utils/pbench/tmp/tm-lite-remote-b.example.com.err
+/var/tmp/pbench-test-utils/pbench/tmp/tm-lite-remote-b.example.com.out
+/var/tmp/pbench-test-utils/pbench/tmp/tm-lite-remote-c.example.com.err
+/var/tmp/pbench-test-utils/pbench/tmp/tm-lite-remote-c.example.com.out
+/var/tmp/pbench-test-utils/pbench/tools-v1-lite
+/var/tmp/pbench-test-utils/pbench/tools-v1-lite/remote-a.example.com
+/var/tmp/pbench-test-utils/pbench/tools-v1-lite/remote-a.example.com/mpstat
+/var/tmp/pbench-test-utils/pbench/tools-v1-lite/remote-b.example.com
+/var/tmp/pbench-test-utils/pbench/tools-v1-lite/remote-b.example.com/__label__
+/var/tmp/pbench-test-utils/pbench/tools-v1-lite/remote-b.example.com/mpstat
+/var/tmp/pbench-test-utils/pbench/tools-v1-lite/remote-b.example.com/node-exporter
+/var/tmp/pbench-test-utils/pbench/tools-v1-lite/remote-c.example.com
+/var/tmp/pbench-test-utils/pbench/tools-v1-lite/remote-c.example.com/__label__
+/var/tmp/pbench-test-utils/pbench/tools-v1-lite/remote-c.example.com/dcgm
+/var/tmp/pbench-test-utils/pbench/tools-v1-lite/remote-c.example.com/pcp
+/var/tmp/pbench-test-utils/pbench/tools-v1-lite/testhost.example.com
+/var/tmp/pbench-test-utils/pbench/tools-v1-lite/testhost.example.com/dcgm
+/var/tmp/pbench-test-utils/pbench/tools-v1-lite/testhost.example.com/mpstat
+=== /var/tmp/pbench-test-utils/pbench/tmp/tm-lite-remote-a.example.com.err:
+ERROR pbench-tool-meister fetch_message -- lost connection to redis server on channel pbench-agent-cli-to-tms
```

Comment:
This is not supposed to happen here, we should have a clean shutdown, but I don't think this code is responsible for this behavior, though it is hard to tell.

Reply:
Maybe we can look at my test environment as my agent tests are not passing completely.